src/model/ScheduleTypeRegistry.cpp

```diff
@@ -619,7 +628,14 @@ namespace model {
       // isStringent = false, we do not enforce NOT having lower / upper limits if our object accepts any.
       // This is user-specified, so user is free to do this
       if (!isCompatible(scheduleType, *scheduleTypeLimits, false)) {
-        result = false;
+        // Skip if user has disabled it
+        if (ScheduleTypeRegistry::instance().enforceScheduleTypeLimits()) {
+          result = false;
+        } else {
+          LOG_FREE(Warn, "checkOrAssignScheduleTypeLimits",
+                   "Schedule '" << schedule.nameString() << "' has a ScheduleTypeLimits named '" << scheduleTypeLimits->nameString()
+                                << "' that is not compatible, but you have turned off enforcement.");
+        }
```

Comment:
If enforce turned off:
* If provided ScheduleTypeLimits is not compatible, still assign but warn.
* If no ScheduleTypeLimits provided, still assign one